core-services/release-controller/_releases/release-ocp-4.8-ci.json

```diff
@@ -83,14 +97,26 @@
       "upgradeFrom": "PreviousMinor",
       "prowJob":{"name":"release-openshift-origin-installer-e2e-gcp-upgrade-4.7-stable-to-4.8-ci"}
     },
+    "upgrade-gcp-ovn-minor":{
+      "interval":"6h",
+      "upgrade":true,
+      "upgradeFrom": "PreviousMinor",
+      "prowJob":{"name":"release-openshift-origin-installer-e2e-gcp-ovn-upgrade-4.7-stable-to-4.8-ci"}
+    },
     "upgrade-azure-minor":{
       "interval":"6h",
       "upgrade":true,
       "upgradeFrom": "PreviousMinor",
       "prowJob":{"name":"release-openshift-origin-installer-e2e-azure-upgrade-4.7-stable-to-4.8-ci"}
     },
+    "upgrade-azure-ovn-minor":{
+      "interval":"6h",
+      "upgrade":true,
+      "upgradeFrom": "PreviousMinor",
+      "prowJob":{"name":"release-openshift-origin-installer-e2e-azure-ovn-upgrade-4.7-stable-to-4.8-ci"}
+    },
     "upgrade-ovirt-minor":{
-      "cron":"0 0,13 * * *",
+      "cron":"0 3,16,22 * * *",
```

Comment:
I would leave the original values here.  These jobs are finicky with their resources and the ovirt team has been handling these.  I suspect they will need to make tweaks when the 4.9 releases are introduced.

Reply:
I think it may be ok - their 4.8 config has to match 4.7 anyway for new stuff.  We can have them come back and follow up.